sdk/dnsresolver/Azure.ResourceManager.DnsResolver/tests/DnsResolverTestBase.cs

```diff
@@ -49,11 +51,11 @@ protected async Task<ResourceGroupResource> CreateResourceGroupAsync()
             return rgOp.Value;
         }
 
-        protected async Task CreateVirtualNetworkAsync(string virtualNetworkName)
+        protected async Task CreateVirtualNetworkAsync()
```

Comment:
Could we optimize this method to make it return `Task<(ResourceIdentifier VNetId, ResourceIdentifier SubnetId)>`?
It is not very convenient to use instance property to store these two ids.

Reply:
fixed